QUARKUS-728.md

```diff
@@ -25,35 +24,13 @@ Test development will focus on
    - OpenShift OSD with UBI8 OpenJDK 17
    - RHEL 9 x86_64 with OpenJDK 17
   
- That could be reflected in the following Jenkins jobs:
+ That could be reflected in the following Jenkins matrix jobs:
```

Comment:
Think the "matrix" part could be omitted - one of the jobs is not matrix, after all :)